### ISSUE TITLE
Allow scanning hidden directories

### ISSUE THREAD

**Issue by ribbybibby** [bridgecrewio/checkov#1292]
It seems like sub-directories that start with a `.` are ignored by checkov: https://github.com/bridgecrewio/checkov/blob/master/checkov/common/runners/base_runner.py#L23.

I can't see a way to workaround this issue, other than copying the contents of the hidden directory to a non-hidden one, or running checkov on the hidden dir separately.

The use case I have in mind is scanning the `.terraform/modules` directory. I appreciate that some people may prefer to exclude this directory, as it may contain modules they aren't responsible for. However, from my perspective, I want to ensure that remote modules adhere to the same standards as the local module.

**Comment by schosterbarak** [comment]
hey @ribbybibby this is possible by editing CKV_IGNORED_DIRECTORIES environment variable that is described here: https://github.com/bridgecrewio/checkov#skipping-directories

**Comment by ribbybibby** [comment]
@schosterbarak I don't think it is. If you look at the code I linked, it's excluding any directories that start with `.` regardless of what `CKV_IGNORED_DIRECTORIES` is set to:
```
    [d_names.remove(d) for d in list(d_names) if d in ignored_directories or d.startswith(".")
```

This is backed up by my observations:
```
$ tree -a
.
├── .terraform
│   └── modules
│       └── hidden
│           └── main.tf
└── not-hidden
    └── modules
        └── hidden
            └── main.tf

6 directories, 2 files

$ cat not-hidden/modules/hidden/main.tf
resource "aws_s3_bucket" "example" {
  bucket = "example"
}

$ cat .terraform/modules/hidden/main.tf
resource "aws_s3_bucket" "example" {
  bucket = "example"
}

$ CKV_IGNORED_DIRECTORIES="node_modules" checkov -c CKV_AWS_52 -d .


       _               _
   ___| |__   ___  ___| | _______   __
  / __| '_ \ / _ \/ __| |/ / _ \ \ / /
 | (__| | | |  __/ (__|   < (_) \ V /
  \___|_| |_|\___|\___|_|\_\___/ \_/

By bridgecrew.io | version: 2.0.180
Update available 2.0.180 -> 2.0.194
Run pip3 install -U checkov to update


terraform scan results:

Passed checks: 0, Failed checks: 1, Skipped checks: 0

Check: CKV_AWS_52: "Ensure S3 bucket has MFA delete enabled"
        FAILED for resource: aws_s3_bucket.example
        File: /not-hidden/modules/hidden/main.tf:1-3

                1 | resource "aws_s3_bucket" "example" {
                2 |   bucket = "example"
                3 | }

```
I would expect to see failed checks for `./terraform/modules/hidden` here.
